docs/permission_template.md

```diff
@@ -0,0 +1,15 @@
+# Permission Template
+
+The permission template is a mechanism in sonar to setup project permissions. The default template will be changed
+during the dogu startup to ensure that the CES_ADMIN group has access to administer new created projects. The following
+permissions will be set (admin codeviewer issueadmin securityhotspotadmin scan user ) this setup can be verified
+(Administration -> Security -> Permisssion Templates). *see setup.json for further details*
+
+![default template overview][assets/default_template_ces_admin_permissions.png]
+
+Apart from project permissions there are some global permission, these permission will be set up during
```

Comment:
That whole text reads quite well :)

Anyhow, since global permissions and project permissions differ so strongly from each other, I'd suggest to move this paragraph into its own file (because this doc specifically focuses on permission templates) or create a harsh distinguishing along the lines of:

> Apart from project permission .... blablabla.... Yet, global permissions (as well as the mapping to the CES admin group) apply to a different scope and will not be discussed in this document.

Reply:
i can do that. I will also adapt the new documentation guideline.